See https://github.com/freeswitch/spandsp/pull/59

--- spandsp-3.0.0.orig/src/Makefile.am
+++ spandsp-3.0.0/src/Makefile.am
@@ -361,6 +361,7 @@ nobase_include_HEADERS = spandsp/ademco_
                          spandsp/private/v42.h \
                          spandsp/private/v42bis.h \
                          spandsp/private/v8.h \
+                         spandsp/private/v80.h \
                          spandsp/expose.h
 
 nodist_include_HEADERS = spandsp.h
