--- a/src/mod/applications/mod_commands/mod_commands.c	2022-06-13 19:58:55.213554487 +0000
+++ b/src/mod/applications/mod_commands/mod_commands.c	2022-06-13 19:59:14.397944099 +0000
@@ -5987,7 +5987,7 @@
 					switch_log_printf(SWITCH_CHANNEL_LOG, SWITCH_LOG_CRIT, "Memory Error!\n");
 					holder.stream->write_function(holder.stream, "-ERR Memory Error!\n");
 				} else {
-					holder.stream->write_function(holder.stream, "%s", json_text);
+					holder.stream->write_function(holder.stream, "%s\n", json_text);
 				}
 				cJSON_Delete(result);
 				switch_safe_free(json_text);
